app/services/events_service.rb

```diff
@@ -0,0 +1,7 @@
+class EventsService
+
+  def self.find_by_zipcode(zipcode)
+    response = Faraday.get("http://eventful.com/json/app_key=#{ENV["CLIENT_ID"]}/events?q=music&l=#{zipcode}")
```

Comment:
this url isn't quite right, but you're on the right track. we don't want to query for music here. and we want to make sure we're hitting the "search" endpoint on Eventful's server - so your url should start with `http://api.eventful.com/json/events/search`